toonz/sources/toonz/xshcellviewer.cpp

```diff
@@ -1921,11 +1921,8 @@ void CellArea::drawSoundTextCell(QPainter &p, int row, int col) {
 
   QFontMetrics metric(font);
 
-  if (text.size() && text != "" && m_viewer->getFrameZoomFactor() <= 50) {
-    text          = text[0];
-    int charWidth = metric.width(text, 1);
-    if (charWidth > nameRect.width()) nameRect.adjust(-2, 0, 4, 0);
-  }
+  int charWidth = metric.width(text, 1);
+  if ((charWidth * 2) > nameRect.width()) nameRect.adjust(-2, 0, 4, 0);
```

Comment:
What do you think of something like this?
```
QFontMetrics metric(font);
if (text.size() && text != "" && m_viewer->getFrameZoomFactor() <= 50) {
	  if (m_viewer->getFrameZoomFactor() <= 20) text = text[0];
	  int charWidth = metric.width(text, 1);
	  if (charWidth > nameRect.width()) nameRect.adjust(-2, 0, 4, 0);
  }

  if (!sameLevel || prevCell.m_frameId != cell.m_frameId)
    p.drawText(nameRect, Qt::AlignLeft | Qt::AlignBottom, text);
```

I did two main things: 
- got rid of the elideText call - I think it is more useful to see part of a letter than a tilde(~).
- If the cell is a certain width, go back to just having one letter.  

This is just my preference.  But I thought I'd throw it out there.

Reply:
Not sure I agree with taking out the elideText call entirely.

That tilde indicates there is more text that isn't displayed. Depending on where its clipped, it may look like there isn't any more text. At least with that visual indicator, you know to hover over it to get the full text if you need it.

Reply:
I see your point.   I'm fine with it this way too.  I have reviewed this and it works fine.  If @shun-iwasawa wants to call Jenkins on this, this is ready.